prime-dotnet-webapi/ViewModels/Health Authority Sites/HealthAuthoritySiteSubmissionViewModel.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.EntityFrameworkCore;
+using Prime.Models;
+
+namespace Prime.ViewModels.HealthAuthoritySites
+{
+    [Keyless]
+    public class HealthAuthoritySiteSubmissionViewModel
+    {
+        public string SiteName { get; set; }
+        public AddressViewModel SiteAddress { get; set; }
+        public string AuthorizedUserFullName { get; set; }
+        public string HealthAuthorityName { get; set; }
+        public string CareType { get; set; }
+        public string PEC { get; set; }
+        public string NewOrExisting
```

Comment:
So site has a field "ActiveBeforeRegistration" not sure if this is what they want displayed since as is this field is already easily deciphered but if the PEC exists or not.

Reply:
NewOrExisting is based on the exitance of a PEC. Will keep that for now as per our discussion.

Reply:
Closing as discussed